.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+### Are you requesting a feature or reporting a bug?
+
+
+
+### What is the current behavior?
+
+
+
+### How would you reproduce this behavior (if this is a bug)?
+
+
+
+### What is the expected behavior?
+
+
+
+### Provide the test code and the tested page URL (if applicable)
```

Comment:
It should be subsection of `How would you reproduce this behavior (if this is a bug)?`